src/index.css

```diff
@@ -11,37 +11,6 @@
 .singlestretch-child {
     height: 100%;
 }
-/* alternative implementations */
```

Comment:
why remove this ?

Reply:
Maybe we could reimplement this with mui grids from the ground up (except for body and the app div)

Reply:
That is old dead code, is it still relevant ?